### ISSUE TITLE
Not compiled

### ISSUE THREAD

**Issue by spa-pal** [s00500/ESPUI#224]
I am trying to compile a "gui" example.

Arduino IDE 1.8.19 replies:
"
C:\Users\PAL\Documents\Arduino\libraries\ESPUI\src\ESPUIclient.cpp: In member function 'uint32_t ESPUIclient::prepareJSONChunk(uint16_t, ArduinoJson6152_0000010::DynamicJsonDocument&, bool)':
C:\Users\PAL\Documents\Arduino\libraries\ESPUI\src\ESPUIclient.cpp:312:25: error: 'ArduinoJson6152_0000010::DynamicJsonDocument' {aka 'class ArduinoJson6152_0000010::BasicJsonDocument<ArduinoJson6152_0000010::DefaultAllocator>'} 
has no member named 'overflowed'
  312 |             if (rootDoc.overflowed())
      |                         ^~~~~~~~~~
"
All libraries are included ( but it's not clear)) ).

Please explain to those who know

**Comment by iangray001** [comment]
I'm not on my dev box, but it could be an issue with the installed ArduinoJson library. Previous versions didn't have that method so are you sure you are using an up to date version of it?

**Comment by MartinMueller2003** [comment]
Yes that is a very old version of ArduinoJson and it does not have this function. Please update to the latest version.
